src/tests/ftest/launch.py

```diff
@@ -1404,18 +1407,18 @@ def install_debuginfos():
     # yum_base.processTransaction(rpmDisplay=yum.rpmtrans.NoOutputCallBack())
 
     # Now install a few pkgs that debuginfo-install wouldn't
-    cmd = ["sudo", "yum", "-y", "--enablerepo=*debug*", "install"]
+    cmd = timeout + ["sudo", "dnf", "-y", "--enablerepo=*debug*", "install"]
```

Comment:
`dnf` shouldn't need a timeout applied.  We haven't seen it hang the way `yum` does.
```suggestion
    cmd = ["sudo", "dnf", "-y", "--enablerepo=*debug*", "install"]
```

Reply:
Fixed